openassessment/xblock/openassessmentblock.py

```diff
@@ -422,6 +423,40 @@ def _(self):
         return i18nService.ugettext
 
     @property
+    def prompts(self):
+        """
+        Return the prompts.
+
+        Initially a block had a single prompt which was saved as a simple
+        string in the prompt field. Now prompts are saved as a serialized
+        list of dicts in the same field. If prompt field contains valid json,
+        parse and return it. Otherwise, assume it is a simple string prompt
+        and return it in a list of dict.
+
+        Returns:
+            list of dict
+        """
+        return create_prompts_list(self.prompt)
+
+    @prompts.setter
+    def prompts(self, value):
```

Comment:
I'm a little confused about what's being done here. Are we trying to dynamically update "prompt" when "prompts" is set? If so, I imagined "prompt" being a deprecated value, that we could read when "prompts" is not set, but that nothing would actually use "prompt" any longer.